### PR TITLE
Custom ModDataChecker for Enderal

### PR DESCRIPTION
**Pull request by Holt59**
See https://github.com/ModOrganizer2/modorganizer-game_gamebryo/pull/17

### CODE REVIEW

File: src/enderalmoddatachecker.h [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    return result;
+  }
+  virtual const FileNameSet& possibleFileExtensions() const override {
+    static FileNameSet result{
+      "esp", "esm", "bsa", ".modgroups"
+    };
+    return result;
+  }
+
+  const GameGamebryo* m_Game;
```
Comment (Al12rs) [inline src/enderalmoddatachecker.h]: Why is this here?

Comment (Holt59) [reply src/enderalmoddatachecker.h]: Because I made a stupid copy/paste :D

**Review by Al12rs — APPROVED** [review]
